src/database/seeders/20190819153902-user.js

```diff
@@ -2,6 +2,17 @@
 module.exports = {
   up: (queryInterface, Sequelize) => {
     return queryInterface.bulkInsert('Users', [
+      {
+        id: '231a136e-a3e2-4911-86f3-63d7db512d3f',
+        firstName: 'line',
+        lastName: 'manager',
+        email: 'linemanager@gmail.com',
+        password: '$2b$07$O0CryUZeuWn0hLsqKcVbvO1ZQbryxV.NJbIAMHqbpkiZqQYjgAD02',
```

Comment:
rebase with develop and follow the pattern

Reply:
Done